public/index.php

```diff
@@ -1,27 +1,9 @@
 <?php
 
 use App\Kernel;
-use Symfony\Component\Debug\Debug;
-use Symfony\Component\HttpFoundation\Request;
 
-require dirname(__DIR__).'/src/.bootstrap.php';
+require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
 
-if ($_SERVER['APP_DEBUG']) {
-    umask(0000);
-
-    Debug::enable();
-}
-
-if ($trustedProxies = $_SERVER['TRUSTED_PROXIES'] ?? $_ENV['TRUSTED_PROXIES'] ?? false) {
```

Comment:
We need to do alot of trusted proxy tests on stg.

Reply:
Flex defaults again. But yes. Should be tested.